articles/sql-database/sql-database-automated-backups.md

```diff
@@ -48,7 +48,13 @@ You can try some of these operations using the following examples:
 
 ## How long are backups kept
 
-Each SQL Database has a default backup retention period between 7 and 35 days that depends on the purchasing model and service tier. You can update the backup retention period for a database on SQL Database server. For more information, see [Change Backup Retention Period](#how-to-change-the-pitr-backup-retention-period).
+All Azure SQL databases (single, pooled, and managed instance databases) have a default backup retention period of  **seven** days. You can [change backup retention period up to 35 days](#how-to-change-the-pitr-backup-retention-period).
+
+> [!NOTE]
+> Starting July 2019, the default backup retention period for DTU-based databases became **seven** days. Previously, the default retention period for DTU-based databases was dependent on service tier:
+>- Basic service tier was **one** week.
```

Comment:
I think we need to just say that starting July 2019, all service tiers have the same default backup retention period of seven days. No need to explain what it was in a Note.

Reply:
sure